src/restore_config.c

```diff
@@ -957,10 +992,18 @@ usage(const char *name)
 	fprintf(stdout, "                      Used to restore to a different namespace.\n");
 	fprintf(stdout, "  -d, --directory <directory>\n");
 	fprintf(stdout, "                      The directory that holds the backup files. Required, \n");
-	fprintf(stdout, "                      unless -i is used.\n");
+	fprintf(stdout, "                      unless -i or --directory-list is used.\n");
 	fprintf(stdout, "  -i, --input-file <file>\n");
 	fprintf(stdout, "                      Restore from a single backup file. Use - for stdin.\n");
-	fprintf(stdout, "                      Required, unless -d is used.\n");
+	fprintf(stdout, "                      Required, unless -d or --directory-list is used.\n");
+	fprintf(stdout, "      --directory-list <path 1>[,<path2[,...]]\n");
+	fprintf(stdout, "                      A comma seperated list of paths to directories that hold the backup files. Required, \n");
+	fprintf(stdout, "                      unless -i or -d is used. The paths may not contain commas.\n");
```

Comment:
I don't think commas are allowed in paths anyway? Not sure if that needs to be said

Reply:
I've never seen it but they are allowed.